data/json/itemgroups/Weapons_Mods_Ammo/nested_ammo.json

```diff
@@ -440,6 +440,17 @@
       { "item": "300blk", "prob": 30, "charges": [ 66, 120 ] }
     ]
   },
+  {
+    "id": "on_hand_30carbine",
+    "type": "item_group",
+    "//": "a collection of ammo that would be found with a loaded gun.",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "30carbine", "prob": 100, "charges": [ 1, 20 ] },
```

Comment:
this should be based on the box size

So you have the count in the actual armor saying it's got a count of 50 (so 50 rounds in a box).

So whatever count the ammo is sold in should be the same across the bullet and this group.